### PR TITLE
fix typos, add config example from app developer manual

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -93,3 +93,35 @@ meta/hooks/config exit with status 10 and message:
 	    error: Unknown config option "tea-in-the-morning"
 
 [snappy fails the install of the app]
+
+### Example: list items and changing config data locally
+
+Running `snappy config SNAPNAME` command will dump the current config of a snap:
```
Comment (elopio) [inline docs/config.md]: maybe print instead of dump ?

File: docs/config.md [hunk 2]
```diff
@@ -93,3 +93,35 @@ meta/hooks/config exit with status 10 and message:
...
+syntax to the snap:
+
+	$ snappy config SNAPNAME > my.yaml
+	$ sed -i 's/listitem1/listitem1-changed/'
+	$ snappy config SNAPNAME my.yaml
+
+This command will fail with a non-zero exit code if the YAML syntax is not
+valid or the config handler of the snap signals an error. 
+
+If it succeeds, the new config will have been applied:
```
Comment (elopio) [inline docs/config.md]: I would change "will have been" for "is".

**Comment by elopio** [comment]
:+1: 
just a couple of comments. Feel free to ignore and merge if you think they don't improve the text.

**Comment by niemeyer** [comment]
Looks good for the time being. We'll want partial configs soon. Sprint topic.

**Pull request merged**